src/tests/homework.spec.js

```diff
@@ -0,0 +1,122 @@
+import { test, expect } from '@playwright/test';
+const { v4: uuidv4 } = require('uuid');
+
+
```

Comment:
tohle jenom organizacni doporuceni. Scenare mohly byt rozdeleny do dvou describe bloku, nebo vsechny v jednom. Tady to zacina be describe bloku a pozdeji mas definovay describe blok, tak mi to prijde mene prehledne.